src/optimize-waterallocation.jl

```diff
@@ -43,7 +43,10 @@ serialize(open(datapath("extraction/waterfromgw$suffix.jld"), "w"), reshape(sol.
 if withreservoirs
     serialize(open(datapath("extraction/captures$suffix.jld"), "w"), reshape(sol.sol[sum(varlens[1:4])+1:end], numreservoirs, numsteps))
 else
-    rm(datapath("extraction/captures$suffix.jld"))
+    storedcaptures = cached_fallback("extraction/captures", () -> false)
+    if storedcaptures != false
+        rm(datapath("extraction/captures$suffix.jld"))
+    end
 end
```

Comment:
Can you explain this change?  It seems like it's saying to delete the file if it can be read, which is what the original did.

Reply:
I believe that if the file is non existent, it will try to delete a file that does not exist and return an error ...

Reply:
Okay, I see that.  But you don't want to take all the time to read the file, if you're just going to delete it.  The way to check if a file exists in julia is `isfile(path)`.